packages/foodie-backend/src/app/user/UserSchema.ts

```diff
@@ -1,11 +1,11 @@
 import Joi from '@hapi/joi';
 
-export const RegisterSchema = Joi.object().keys({
+export const UserSchema = Joi.object().keys({
```

Comment:
Will this be used for validation the four http medthds , POSST, GET,PUT, DELETE. I suggest that is wrapped in an fucntion following ssp

Reply:
this would be use to define the schema of the user